devdoc/userdoc/JiraGuidelines.md

```diff
@@ -0,0 +1,43 @@
+# JIRA Creation Guidelines
+
+This is a guide to the type of information that is desired in JIRA tickets.
+
+# For bugs / improvement requests on any existing features or docs: 
+   - Provide a screenshot when it’s applicable.  Not just of the offending line or excerpt, but of the entire application window.
+      - This is especially true for issues related to ECL Watch.
+   - Provide a zap file if there is one.
+   - Build version of the environment you're using.
+   - URL to environment.
+   - URL to workunit.
+   - Which browser and its version.
+   - Steps for reproducing.
+      - Laid out in an easy to understand format.  
+      - Either a numbered set of steps or even something simple as:
+         - Open Browser > Navigate to here > then click this tab 
+   - Any artifacts that are relevant to the recreation of the issue.
+      - These include input files, code, pkgmap contents, etc.
+   - Type of environment where the HPCC Platform is running.
+      - Bare-metal environment, VM, container?
+      - Details on the systems used to start these such as:
+         - Operation system 
+         - Minikube or Docker Desktop
+         - Virtual Box
+   - Steps taken to install the platform if this is related to installation and init.
+   - If this is a bare-metal build, where did you get your package and what is the md5sum of this package?
+   - Include stack traces and a copy of the core file if possible.
```

Comment:
I don't think we generally want core files attached - they may contain pii, and can be GBs in size.